src/en/docs/usage-basic-overview.md

```diff
@@ -5,24 +5,26 @@ priority: 2
 translationKey: usage-basic-overview
 ---
 
-{% optimizedImage './timeline.png', 'Screenshot of Photoview', 'class="block w-4/5 mx-auto my-6"' %}
+{% optimizedImage '../../assets/images/timeline.png', 'Screenshot of Photoview', 'class="block w-4/5 mx-auto my-6"' %}
```

Comment:
Image is not loaded now because of the wrong path (extra `..`). I think that it is better to use the path `src/assets/images/screenshot-photos.png` - it already exists in. the repo, so please remove your duplicated `src/assets/images/timeline.png` from the PR and use the path to the existing screenshot as I provided iit

Reply:
It was not clear to me that starting the patch with "src" makes it relative to the root of the repo. I've made the change now and hope it works.